java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java

```diff
@@ -19,15 +19,104 @@
 
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
-import org.openqa.selenium.StandardSeleniumTests;
-import org.openqa.selenium.devtools.DevToolsTests;
+import org.junit.runners.model.InitializationError;
 
-
-@RunWith(Suite.class)
-@Suite.SuiteClasses({
-    StandardSeleniumTests.class,
-    ChromeOptionsFunctionalTest.class,
-    DevToolsTests.class
-})
+@RunWith(ChromeDriverTests.ChromeTestRunner.class)
 public class ChromeDriverTests {
```

Comment:
The way we're working on making this work is to avoid test suites, and instead use `java_selenium_test_suite` in our `BUILD.bazel` files. This creates a target per test, and so allows us to shard them more efficiently. It's defined in `//java:defs.bzl`.

That takes a `browser` attribute: https://github.com/SeleniumHQ/selenium/blob/master/java/private/test.bzl#L29 which I suspect you want to set to `["chrome", "edge"]`